src/App.js

```diff
@@ -1,10 +1,75 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+import Title from './Title';
+import Candidate from './Candidate';
```

Comment:
이렇게 직접 만든 컴포넌트를 import할때는 개행으로 구분해주시면 좀 더 보기 좋을 것 같아요:)

```suggestion
import React, {useState, useEffect} from 'react';
import axios from 'axios';
import styled from 'styled-components';

import Title from './Title';
import Candidate from './Candidate';
```

Reply:
구분 하도록 하겠습니다 !!